packages/react/src/lib/renderComponent.tsx

```diff
@@ -16,6 +16,8 @@ import {
   PropsWithVarsAndStyles,
   State,
   ThemePrepared,
+  ProviderContextPrepared,
```

Comment:
would expect these two types to be declared in `../types`, not in theme types, as those are not dependent on Stardust's theme semantics